### PR TITLE
Feat: Base docker image

### PR DESCRIPTION
**Pull request by Meldiron**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -9,76 +9,18 @@ COPY composer.json /usr/local/src/
...
-FROM php:8.3.7-cli-alpine3.19 as final
+FROM open-runtimes/base:0.1.0 as final
 
 ARG OPR_EXECUTOR_VERSION
 ENV OPR_EXECUTOR_VERSION=$OPR_EXECUTOR_VERSION
 
-LABEL maintainer="team@appwrite.io"
-
-ENV DOCKER_CONFIG=${DOCKER_CONFIG:-$HOME/.docker}
-ENV DOCKER_API_VERSION=1.43
```
Comment (PineappleIOnic) [inline Dockerfile:50]: Do we not need these `DOCKER` environment variables?

File: Dockerfile [hunk 2]
```diff
@@ -9,76 +9,18 @@ COPY composer.json /usr/local/src/
...
 COPY ./app /usr/local/app
 COPY ./src /usr/local/src
 
 # Extensions and libraries
 COPY --from=composer /usr/local/src/vendor /usr/local/vendor
-COPY --from=swoole /usr/local/lib/php/extensions/no-debug-non-zts-20230831/swoole.so /usr/local/lib/php/extensions/no-debug-non-zts-20230831/
-
-RUN echo extension=swoole.so >> /usr/local/etc/php/conf.d/swoole.ini
-
-EXPOSE 80
```
Comment (PineappleIOnic) [inline Dockerfile:81]: Don't we need this `EXPOSE 80`?

File: Dockerfile (hunk 1 above)
Comment (Meldiron) [reply Dockerfile:50]: They are set in base docker image: https://github.com/open-runtimes/docker-base/pull/1/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R35-R36

File: Dockerfile (hunk 2 above)
Comment (Meldiron) [reply Dockerfile:81]: Base image defines that: https://github.com/open-runtimes/docker-base/pull/1/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R62

Plus tests pass, makes me confident it works

**Review by PineappleIOnic — APPROVED** [review]

File: Dockerfile [hunk 3]
```diff
@@ -9,76 +9,18 @@ COPY composer.json /usr/local/src/
...
-  git clone --depth 1 --branch $PHP_SWOOLE_VERSION https://github.com/swoole/swoole-src.git && \
-  cd swoole-src && \
-  phpize && \
-  ./configure --enable-sockets --enable-http2 --enable-openssl --enable-swoole-curl && \
-  make && make install && \
-  cd ..
-
 # Executor
-FROM php:8.3.7-cli-alpine3.19 as final
+FROM open-runtimes/base:0.1.0 as final
```
Comment (Meldiron) [inline Dockerfile:13]:
```suggestion
FROM openruntimes/base:0.1.0 as final
```

**Pull request merged**